### DISCUSSION TITLE
Snap Deal Performance

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#8127] [Storage Provider]
Starting this discussion to have a place for SPs to share the [Snap Deal ](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0019.md)performance with their system! 

| Hardware Spec  (miner/worker, GPU/CPU/RAM) |  sector size   | RU - UpdateReplica (m's'')  | PR2 - ProveReplicaUpdate (m's'')  | CUDA/OpenCL | C2 - ProveCommit (m's'')   | 
|---|---|---|---|---|---|
|   |   |   |   |   |  |

**Comment by Reiers** [comment]
Hardware Spec (miner/worker, GPU/CPU/RAM) | sector size | RU - UpdateReplica (m's'') | PR2 - ProveReplicaUpdate (m's'') | CUDA/OpenCL | C2 - ProveCommit (m's'')
-- | -- | -- | -- | -- | --
| DUAL 3090s/3975WX/512GB RAM | 32GiB | 5m38s | 7m52s | CUDA | 6m57s |

**Comment by benjaminh83** [comment]
First benchmark done on my calibration network miner with one external sealing worker: 

| Hardware Spec (miner/worker, GPU/CPU/RAM)      | sector size        | RU - UpdateReplica (m's'')  | PR2 - ProveReplicaUpdate (m's'') | CUDA/OpenCL | C2 - ProveCommit (m's'')
|:-----:|:------:|:-----:|:-----:|:-----:|:-----:|
| EPYC 7F72, 256GiB mem + 20GiB SWAP, 2x RTX 2080ti | 64GiB | 9m19s | 19m0s | CUDA | 16m10s |

**Comment by Meest-NAM** [comment]
I'm Meest NAM
|Hardware Spec (miner/worker, GPU/CPU/RAM)|sector size|RU - UpdateReplica (m's'')|PR2 - ProveReplicaUpdate (m's'')|CUDA/OpenCL|C2 - ProveCommit (m's'')|
|---|---|---|---|---|---|
|EPYC 7502, 512GB mem + Noswap, RTX3080|64GiB|12m 59s|23m 13s|CUDA|18m 24s|

**Comment by SBudo** [comment]
<html>
<body>
<br class="Apple-interchange-newline">

Hardware Spec (miner/worker, GPU/CPU/RAM) | sector size | RU - UpdateReplica (m's'') | PR2 - ProveReplicaUpdate (m's'') | CUDA/OpenCL | C2 - ProveCommit (m's'')
-- | -- | -- | -- | -- | --
EPYC 7302, 256GB RAM (no swap), Tesla T4 | 32GiB | 3m 37s | 33m 11s | CUDA | 26m 36s


</body>
</html>

**Comment by donkabat** [comment]
Hardware Spec (miner/worker, GPU/CPU/RAM) | sector size | RU - UpdateReplica (m's'') | PR2 - ProveReplicaUpdate (m's'') | CUDA/OpenCL | C2 - ProveCommit (m's'')
-- | -- | -- | -- | -- | --
RTX3090/7252/256GB RAM (no swap) | 32GiB | 3m 33s | 14m 51s|  CUDA | -
RTX3090/7343/256GB RAM (no swap) | 32GiB | 2m 38s | 12m 32s | CUDA | -
RTX2080Ti/5800x/80GB RAM (no swap) | 32GiB | 3m 40s | - | CUDA | -
